setup.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
For portability, possibly use '#!/usr/bin/env bash' in case bash is in /usr/bin or somewhere else entirely random?